tests/containers/rootless_podman.pm

```diff
@@ -154,7 +154,8 @@ sub verify_userid_on_container {
     $cid = script_output "podman run -d --rm --name test2 --user 1000 $image sleep infinity";
     $cid = check_bsc1200623($cid);
     my $id = $start_id + $huser_id - 1;
-    validate_script_output "podman top $cid user huser", sub { /1000\s+${id}/ };
+    my $huser_name = script_output "echo \$USERNAME";
```

Comment:
You can use `'` to prevent variable expansion in strings:
```suggestion
    my $huser_name = script_output 'echo $USERNAME';
```
However, I'd advise against relying on the value of the `USERNAME` variable, as I am not sure if that's a standard value (it's e.g. not set in the fish shell). I'd suggest to use `$(whoami)` instead.

Reply:
That's a good point, I've however, almost always assumed that USERNAME was as standard as UID on most systems. Updated.

Reply:
> Why'd you use command substitution there though? `$(whoami)` I don't know much perl but going by the function name, it looks like it'll return the stdout from whatever command being run as arg.

Because I'm dumb :facepalm: … `$(whoami)` makes absolutely no sense and `script_run 'whoami';` is the correct form. I guess it's time for another :coffee: